tools/check-kernel-config.sh

```diff
@@ -8,15 +8,28 @@ function get_kernel_version() {
   printf "%03d%03d%03d" "${p[0]}" "${p[1]}" "${p[2]}"
 }
 
+function get_rh_version() {
+  if [ ! -f /etc/redhat-release ] ; then
+     echo "000000000"
```

Comment:
```suggestion
     printf "000000000"
```

`echo` adds a trailing `\n` and sometimes you don't want that :)

```sh
$ echo "000" | hexdump
0000000 30 30 30 0a
0000004
```

Reply:
Thank you for the lesson ! :)